src/extension.ts

```diff
@@ -225,6 +232,13 @@ const startsWithF = (input: string, start: number, end: number): boolean => {
   }
   return starts;
 };
+const startsWithROrU = (input: string, start: number, end: number): boolean => {
+  let starts = false;
+  if (start > 0) {
+    starts = input[start - 1] === "r" || input[start - 1] === "u";
```

Comment:
Hello. Thank you for the PR. In python3 (this plugin is valid for 3.6 onwards) 'u' is not valid. We will need to check for 'b' which would be for bytes.

Could you please add test for this condition too.

I'll get to this in a few days and/or add tests, once I have access to system. Thanks.